module/VuFindAdmin/src/VuFindAdmin/Controller/TagsController.php

```diff
@@ -71,7 +71,6 @@ public function homeAction()
     {
         $view = $this->createViewModel();
         $view->setTemplate('admin/tags/home');
-        $view->statistics = $this->getTable('resourcetags')->getStatistics(true);
```

Comment:
I think this is related to the social stats and probably does not need to be removed.